webapp/src/pages/Dashboard.tsx

```diff
@@ -85,6 +85,8 @@ const Dashboard = () => {
       {isPipelineSelected(pipeline) && (
         <PreviewCard
           title="Performance Analysis"
+          to={`/${jobId}/performance_analysis${searchString}`}
+          linkButtonText="Compare models"
           description={
```

Comment:
Can you reuse `performanceAnalysisDescription` from its page to make sure they stay in sync?
```suggestion
          description={performanceAnalysisDescription}
```

Reply:
Sure!. I can do it